src/client/dfs/dfs.c

```diff
@@ -1220,11 +1245,6 @@ dfs_mount(daos_handle_t poh, daos_handle_t coh, int flags, dfs_t **_dfs)
 	if (dfs == NULL)
 		D_GOTO(err_prop, rc = ENOMEM);
 
-	dfs->use_dtx = false;
-	d_getenv_bool("DFS_USE_DTX", &dfs->use_dtx);
```

Comment:
There's a command line option in NLT which sets this env var, so there are three or four lines in that file that should be removed as well.